community/nodejs-current/APKBUILD

```diff
@@ -1,27 +1,20 @@
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
 # Maintainer: Jose-Luis Rivas <ghostbar@riseup.net>
 #
-# secfixes:
-#   9.10.0-r0:
-#     - CVE-2018-7158
-#     - CVE-2018-7159
-#     - CVE-2018-7160
-#   9.2.1-r0:
-#     - CVE-2017-15896
-#     - CVE-2017-15897
-#
 pkgname=nodejs-current
 # The current stable version, i.e. non-LTS.
-pkgver=9.11.1
-pkgrel=4
+
+pkgver=10.10.0
+pkgrel=1
+
 pkgdesc="JavaScript runtime built on V8 engine - current stable version"
 url="https://nodejs.org/"
 arch="all"
 license="MIT"
 depends="ca-certificates"
 depends_dev="libuv"
 # gold is needed for mksnapshot
-makedepends="$depends_dev python2 openssl1.0-dev zlib-dev libuv-dev linux-headers
+makedepends="$depends_dev python2 openssl1.1-dev zlib-dev libuv-dev linux-headers
```

Comment:
`openssl1.1-dev` is in testing repo and could not be used in community

Reply:
as in #5558 the correct package is openssl-dev

Reply:
`openssl1.1-dev` has been renamed to `openssl-dev` and moved to main.